Sources/Subs-Attachments.php

```diff
@@ -1126,11 +1140,14 @@ function getAttachMsgInfo($attachID)
  */
 function getAttachsByMsg($msgID)
 {
-	global $context, $modSettings, $smcFunc, $user_info;
+	global $context, $modSettings, $smcFunc, $user_info, $cacheMsgID;
 	static $attached = array();
 
 	if (!isset($attached[$msgID]))
 	{
+		if (empty($cacheMsgID[$msgID]))
+			prepareAttachsByMsg ([$msgID]);
+/*
```

Comment:
I know our minimum version supports short syntax for arrays [] but to keep some normalization and consistency with the rest of the code please use the old array format